data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/JacksonEvent.java

```diff
@@ -279,15 +280,6 @@ private String trimKey(final String key) {
         return trimmedLeadingSlash.endsWith(SEPARATOR) ? trimmedLeadingSlash.substring(0, trimmedLeadingSlash.length() - 2) : trimmedLeadingSlash;
     }
 
-    private boolean isNumeric(final String str) {
```

Comment:
I'd probably keep `isNumeric` and make this call `StringUtils.isNumeric`. This way, there is one approach within this class. But, this is still great. Thanks!